courses/machine_learning/deepdive2/structured/solutions/3c_bqml_dnn_babyweight.ipynb

```diff
@@ -0,0 +1,1152 @@
+{
```

Comment:
<p>Remove this cell: We can assume prior knowledge of the basic concepts covered in this cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/670/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>This has never been done before in any other lab and the documentation is not public.</p>